src/card.rs

```diff
@@ -211,7 +211,7 @@ pub struct Card {
     pub toughness: Option<String>,
 
     /// The type line of this card.
-    pub type_line: String,
+    pub type_line: Option<String>,
```

Comment:
why did you change these to option? Is the because of the weird double faced cards from the "heads I win, tails you lose" cards?

Reply:
the parsing of the bulk files containing all cards/one print per card etc. failed because some cards are in fact missing/dont have a type line. I forgot which ones were the problem, but I think it was something from an un set

Reply:
never mind, it’s probably because of heads & tails, the first card that threw an error was kraks thumb

Reply:
Yeah, I have never been able to find a solution for those cards I like, every card has those fields except for the 3 double faced cards in that deck, so I really wanted to avoid the unergonomic change but I haven't been able to find a better solution yet so maybe I should surrender to the options?

If so, can you document that these new option fields are present in every card except the 3 double sided ones (and possibly link the cards in the docs)